### PR TITLE
[CUBRIDQA-78] add analyze core file and generate core json file

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

finished the first draft of core info json file generation

### CODE REVIEW

File: CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
+package com.navercorp.cubridqa.common.coreanalyzer;
```
Comment (fanzaiqiang) [inline CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java]: Move MergeTemplate to com.navercorp.cubridqa.common.

File: CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+		}
+		
+		
+		File outFile = new File(outputFile);
+		if(!outFile.exists()){
+			showHelp("The template file" + outFile.getAbsolutePath() + " does not exist!", options);
+			return;
+		}
+		
+		String templateFileContents = com.navercorp.cubridqa.common.CommonUtils.getFileContent(templateFile);
```
Comment (fanzaiqiang) [inline CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java]: Try to use "import com.navercorp.cubridqa.common.CommonUtils;"

File: CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+				String fDataContent = com.navercorp.cubridqa.common.CommonUtils.getFileContent(fdata.getAbsolutePath());
+				com.navercorp.cubridqa.common.CommonUtils.replace(templateFileContents, key, fDataContent);
+			}else{
+				com.navercorp.cubridqa.common.CommonUtils.replace(templateFileContents, key, value);
+			}
+		}
+		
+	}
+
+	public static String readJsonTemplate(String templateFilePath) throws IOException {
```
Comment (fanzaiqiang) [inline CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java]: Remove this method readJsonTemplate().

File: CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+		String templateContents = com.navercorp.cubridqa.common.CommonUtils.getFileContent(templateFilePath);
+		return templateContents;
+	}
+	
+	private static void showHelp(String error, Options options) {
+		if (error != null) {
+			System.out.println("Error: " + error);
+		}
+		HelpFormatter formatter = new HelpFormatter();
+		formatter.printHelp("MergeTemplate.sh <-t|--template-file> template-file <-d|--data> data-file <-o|--out> out-file", options);
```
Comment (fanzaiqiang) [inline CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java]: Please correct:
--data 
--out

File: CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+		System.out.println();
+	}
+
+	public static String escape(String string) {
```
Comment (fanzaiqiang) [inline CTP/common/src/com/navercorp/cubridqa/common/coreanalyzer/MergeTemplate.java]: Move below to CommonUtils.java
public static String escape(String string)

And change 'escape' to 'genJson'

**Comment by supler** [comment]
@fanzaiqiang 

update codes accordingly.

**Comment by fanzaiqiang** [comment]
@supler 

Please commit it. Thanks.

**Pull request merged**